pkg/apis/operator/v1alpha1/install_types.go

```diff
@@ -44,8 +44,8 @@ type Install struct {
 type InstallationScope string
```

Comment:
Add godocs for `InstallationScope`, `InstallationScopeNamespaceScoped`, and `InstallationScopeClusterScoped`